script/deploys/DeployPhaseOnePointFive.s.sol

```diff
@@ -126,11 +126,12 @@ contract DeployPhaseOnePointFiveScript is Script {
     }
 
     function setUpAdmins(address _admin) internal {
-        liquidityPool.updateAdmin(_admin, true);
-        regulationsManager.updateAdmin(_admin, true);
-        membershipManager.updateAdmin(_admin, true);
-        membershipNFT.updateAdmin(_admin, true);
-        nftExchange.updateAdmin(_admin);
+        // DEPRECATED
+        // liquidityPool.updateAdmin(_admin, true);
```

Comment:
This refactor broke many of our previous deployment scripts. I just commented them out for now for V and I to handle later in our test refactor project.